packages/core/src/tools/readBytesFromStream.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { isIE } from './browserDetection'
+import { readBytesFromStream } from './readBytesFromStream'
+
+describe('readBytesFromStream', () => {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏some tests in `networkErrorCollection.spec` could be more suited here now

Reply:
Unit tests in `networkErrorCollection.spec` should be specific to network collection logic and tests in `readBytesFromStream.spec` should be specific to byte read logic.
By having a quick look `does not yield an unhandled rejection error if the cancel promise is rejected` seems to be misplaced to me. is it the case? and if yes, could other ones be misplaced as well?

Reply:
All tests under `describe('computeFetchResponseText'` focus on testing logic from `readBytesFromStream` (especially as we do not test our fallback cases). 

Are you suggesting we remove all of them and add the missing ones to `readBytesFromStream.spec`?

Reply:
I'd like to keep the same level of coverage but have the tests as close as possible of the logic they are testing.
Since you extracted the logic, you should extract and adjust the corresponding tests as well.
Does it make sense?